### PR TITLE
Error page

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #37 #50

### CODE REVIEW

File: src/text.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = {
     homePage: 'Home Page',
     english: 'English',
     arabic: 'Arabic',
+    error: 'Bad request !!',
```
Comment (jwld) [inline src/text.js]: Might be good to have other errors too (e.g. `Not Found`)

Also maybe drop the exclamation marks?

**Comment by m4v15** [comment]
You should address this issue #50 in this PR

And also I would say make an 'error' layout, as well as an 'error' view, and then pass the view some details about the error - like in the handlebars workshop here:
https://github.com/foundersandcoders/express-handlebars-workshop/blob/solution/src/controllers/error.js

I guess the 'layout' part isn't 100% necessary in our case, but *definitely* should have a generic error page and pass it different variables whenever we render it.

**Comment by m4v15** [comment]
Also need to do it in the language check middleware as well:
https://github.com/foundersandcoders/OTP-Data-Entry/blob/error-handling/src/middleware/langError.js#L9

**Comment by Suuuuha** [comment]
@m4v15 @JWLD

File: src/controllers/places/modify_content.js [hunk 2]
```diff
@@ -62,10 +65,13 @@ module.exports = (req, res) => {
   };
   Request(reqOptions, (error, apiResponse, apiResponseBody) => {
     if (error) {
-      res.status(400).send('server error');
+      res.render('error');
```
Comment (jwld) [inline src/controllers/places/modify_content.js]: Does this one need some parameters passing to it?

Comment (m4v15) [reply src/controllers/places/modify_content.js]: @JWLD maybe @Suuuuha so she sees this? Not sure actually if that helps

**Review by jwld — APPROVED** [review]

**Review by m4v15 — APPROVED** [review]

**Pull request merged**